src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -72,7 +87,7 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
         {
             if ( from == null || to == null || actual == null)
                 throw new ArgumentException( "Cannot compare using a null reference", "actual" );
```

Comment:
I totally overlooked that it was preexisting code. @JustinRChou unless you really feel like doing it now, why don't we do a separate PR after this merges?